### PR TITLE
Add simplified collision boxes

### PR DESCRIPTION
**Pull request by TheCoolKidsTable**
This PR adds simplified collision boxes for the robot model. This was achieved using the tool provided by bitbots https://github.com/bit-bots/simplify_urdf_collision.

**Optimal bounding boxes generated:**
image

### CODE REVIEW

**Comment by xXBenjaminYoungXx** [comment]
Needs to be tested (Perhaps use of simple controller). I might also add more comments where necessary then good to go!

**Review by JosephusPaye — APPROVED** [review]
I think we can merge now. Comments can come in a follow up PR.

File: worlds/kid.wbt [hunk 1]
```diff
@@ -89,6 +89,9 @@ DEF BLUE_1 NUgus {
   rotation 0 0 1 3.1415
   controller "nugus_controller"
   controllerArgs [
-    "32"
+    # TCP port to listen on
```
Comment (ysims) [inline worlds/kid.wbt:92]: Why is this in this PR?

**Review by ysims — CHANGES_REQUESTED** [review]
@Bidski put in extra things maybe in the wrong branch?

File: worlds/kid.wbt (hunk 1 above)
Comment (JosephusPaye) [reply worlds/kid.wbt:92]: Was needed to test, and it doesn't seem to be in main.

Comment (JosephusPaye) [reply worlds/kid.wbt:92]: I think it would be easier to add here and merge. But we could move it to a separate PR.

**Review by ysims — APPROVED** [review]
I guess 🤷

**Pull request merged**